localstack-core/localstack/services/cloudformation/provider_utils.py

```diff
@@ -78,6 +85,18 @@ def _keys_pascalcase_to_lower_camelcase(obj):
     return _recurse_properties(model, _keys_pascalcase_to_lower_camelcase)
 
 
+def keys_lower_camelcase_to_pascalcase(model: dict) -> dict:
```

Comment:
nit: I notice this function is not used in the companion PR. Can you give an example of its usage in the PR description?